apps/remix-ide/src/app.js

```diff
@@ -362,7 +362,7 @@ class AppComponent {
             } catch (e) {
               console.log(e)
             }
-            if (params.code) {
+            if (params.code && (!params.activate || params.activate==='solidity')) {
```

Comment:
i think the issue is also that we need to show the activated plugin.
also I think `activate` is a comma separated list of plugins

Reply:
@yann300 should be fine now

Reply:
can you checl that if you load https://remix.ethereum.org/?#activate=starkNet_compiler&code=IyBEZWNsYXJlIHRoaXMgZmlsZSBhcyBhIFN0YXJrTmV0IGNvbnRyYWN0LgolbGFuZyBzdGFya25ldAoKZnJvbSBzdGFya3dhcmUuY2Fpcm8uY29tbW9uLmNhaXJvX2J1aWx0aW5zIGltcG9ydCBIYXNoQnVpbHRpbgoKIyBEZWZpbmUgYSBzdG9yYWdlIHZhcmlhYmxlLgpAc3RvcmFnZV92YXIKZnVuYyBiYWxhbmNlKCkgLT4gKHJlcyA6IGZlbHQpOgplbmQKCiMgSW5jcmVhc2VzIHRoZSBiYWxhbmNlIGJ5IHRoZSBnaXZlbiBhbW91bnQuCkBleHRlcm5hbApmdW5jIGluY3JlYXNlX2JhbGFuY2V7CiAgICAgICAgc3lzY2FsbF9wdHIgOiBmZWx0KiwgcGVkZXJzZW5fcHRyIDogSGFzaEJ1aWx0aW4qLAogICAgICAgIHJhbmdlX2NoZWNrX3B0cn0oYW1vdW50IDogZmVsdCk6CiAgICBsZXQgKHJlcykgPSBiYWxhbmNlLnJlYWQoKQogICAgYmFsYW5jZS53cml0ZShyZXMgKyBhbW91bnQpCiAgICByZXR1cm4gKCkKZW5kCgojIFJldHVybnMgdGhlIGN1cnJlbnQgYmFsYW5jZS4KQHZpZXcKZnVuYyBnZXRfYmFsYW5jZXsKICAgICAgICBzeXNjYWxsX3B0ciA6IGZlbHQqLCBwZWRlcnNlbl9wdHIgOiBIYXNoQnVpbHRpbiosCiAgICAgICAgcmFuZ2VfY2hlY2tfcHRyfSgpIC0+IChyZXMgOiBmZWx0KToKICAgIGxldCAocmVzKSA9IGJhbGFuY2UucmVhZCgpCiAgICByZXR1cm4gKHJlcykKZW5k&optimize=false&runs=200&evmVersion=null&version=soljson-v0.8.7+commit.e28d00a7.js

is the startnet plugin activated and displayed?

Reply:
it only activates it. but this is live, my fix is not deployed yet...